modules/ossm-excluding-individual-pods-from-cluster-wide-mesh-cli.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-excluding-individual-pods-from-cluster-wide-mesh-cli_{context}"]
+= Excluding individual pods from a cluster-wide mesh using the CLI
+
+Pods receive sidecar injection when they exist in a namespace that matches the label selector defined in the `ServiceMeshMemberRoll` resource and the pods have the `sidecar.istio.io/inject` annotation. 
+
+Individual pods are excluded from sidecar injection when they do not have the annotation.
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator.
+* You have deployed a `ServiceMeshControlPlane` resource and created a `ServiceMeshMemberRoll` resource.
+* You are logged in as a user with the `cluster-admin` role. If you use {product-dedicated}, you are logged in as a user with the `dedicated-admin` role.
+
+. Procedure
+
+. Log in to the {product-title} CLI.
+
+. Edit the deployment configuration.
++
+[source,terminal]
+----
+$ oc edit deployment -n <namespace> <deploymentName>
+----
+
+. Modify `spec.template.metadata.annotations` field to include the `sidecar.istio.io/inject: 'true'` annotation.
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  labels:
+    app: nginx
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      annotations:
```

Comment:
@jewertow should line 47 show the annotation as being set to false or should it show the annotation being set to true with an updated bullet 1 that says something like "Deleting this annotation prevents the pod from being accessible in a cluster-wide mesh."